dist/self-service-reasons/1.0.2/cancellation-reasons_nl.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "annuleerForm",
```

Comment:
confirm that name needs to change for other languages as well or change to "cancelForm"